src/hooks/useUpdateEffect.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useFirstMountState } from "./useFirstMountState";
+
+export const useUpdateEffect: typeof useEffect = (effect, deps) => {
```

Comment:
dublicate

Reply:
`hooks.ts` was removed in favor of separate hook-per-file